README.md

```diff
@@ -79,18 +79,24 @@ kogitoapp.app.kiegroup.org/example-quarkus created
 oc delete kogitoapp example-quarkus
 ```
 
-## Development
+## Development 
 
-Change log level at runtime w/ the `DEBUG` environment variable. e.g. -
+Change log level at runtime with the `DEBUG` environment variable. e.g. -
```

Comment:
Could it not have a command where the user is able to o that by cut/past and running it?